### ISSUE TITLE
[chapters] Western Massachusetts 

### ISSUE THREAD

**Issue by dauwhe** [jquery-foundation/standards#57]
There's some interest out here in the Pioneer Valley.... Greenfield, Northampton, Amherst, Springfield... I'd be happy to play a role.

**Comment by bkardell** [comment]
@dauwhe do you have a meetup to partner with already?

**Comment by dauwhe** [comment]
No, but I have a friend (who does non-profit tech) who's very interested, too.

**Comment by akhoury** [comment]
+1
